src/helm-frontend/src/services/getSchema.ts

```diff
@@ -1,16 +1,20 @@
-import type Schema from "@/types/Schema";
-import getBenchmarkEndpoint from "@/utils/getBenchmarkEndpoint";
-import getVersionBaseUrl from "@/utils/getVersionBaseUrl";
+import { parse } from "yaml";
 
-export function getSchemaJsonUrl(): string {
-  return getBenchmarkEndpoint(`${getVersionBaseUrl()}/schema.json`);
-}
+import type Schema from "@/types/Schema";
+import getBenchmarkSuite from "@/utils/getBenchmarkSuite";
+import getBenchmarkRelease from "@/utils/getBenchmarkRelease";
 
 export default async function getSchema(signal: AbortSignal): Promise<Schema> {
   try {
-    const resp = await fetch(getSchemaJsonUrl(), { signal });
+    const resp = await fetch(
```

Comment:
Why are we changing this back from what was in main?

Reply:
This was a relic of my forked codebase not being rebased with your changes. I'll fix it by reverting to your changes.